sw/airborne/firmwares/rotorcraft/autopilot_arming_yaw.h

```diff
@@ -196,6 +214,10 @@ static inline void autopilot_arming_check_motors_on(void)
     }
   } else {
     autopilot.arming_status = AP_ARMING_STATUS_KILLED;
+    if (kill_switch_is_on()) {
+      autopilot_check_motor_status = STATUS_MOTORS_RC_KILLED;
```

Comment:
you need to set the `motor_kill_time` before entering this state

Reply:
This is done in the function `autopilot_arming_set()`. Is that also ok with you?